bench/README.md

```diff
@@ -18,40 +28,243 @@ pip3 install -r bench/requirements.txt
+    ```
+    `<workers-count>`- number of machine.
+
+
+To run write/read load tests
+
+* Open the docker-compose.yml and add a new `command` for `master` and `worker`.  
+    e.g.: for `master`
+    ```yml
+    command: 
+      - "--headless"
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: 
+      - "--headless" 
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine. `Number of users should be equal to number of workers for write tests.`
+
+
+## Web UI
+
+To run write load tests
+
+* Open the docker-compose.yml and add a new `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
+    ```yml
+    command: 
+      - "WriteModelLoadTest" 
+    ```
+    e.g.: for `worker`:
+    ```yml
+    command: 
+      - "WriteModelLoadTest" 
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine. `Number of users should be equal to number of workers for write tests.`
+
+To run read load tests
+
+* Open the docker-compose.yml and add a new `command` for `master` and `worker`.  
+    <br> e.g.: for `master`
+    ```yml
+    command:
+      - "ReadModelLoadTest" 
+    ```
+    e.g.: for `worker`: 
+    ```yml
+    command: 
+      - "ReadModelLoadTest" 
+    ```
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine.
+
+To run write/read load tests
+
+* Run docker-compose.yml
+    ```bash
+    docker-compose up --scale worker=<workers-count>
+    # The workers run your Users and send back statistics to the master. The master instance doesn't run any Users itself. Both the master and worker machines must have a copy of the locustfile when running Locust distributed.
+    ```
+    `<workers-count>`- number of machine. `Number of users should be equal to number of workers for write tests.`
+
+Then you can open `http://localhost:8089/` and launch the tests from the browser.
+
+# Running tests against local pool
```

Comment:
What is the difference between this section and `Run` section? Is seems the only difference is in the configs (docker endpoints in one case and some generic node URLs in another)? Please get rid of that redundancy.